### PR TITLE
New version that's not vulnerable to code injection and has way less deps and stuff

### PR DESCRIPTION
**Pull request by smolck**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,20 +1,16 @@
+cargo-features = ["strip"]
```
Comment (gpanders) [inline Cargo.toml:1]: NSFW warning

File: src/tag_search.rs [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    // pub _tag: &'a str, // TODO(smolck)
+}
+
+impl Ord for Tag<'_> {
+    fn cmp(&self, other: &Self) -> std::cmp::Ordering {
+        self.name.cmp(&other.name)
+    }
+}
+
+impl<'a> Tag<'a> {
```
Comment (gpanders) [inline src/tag_search.rs:16]:
```suggestion
i'm'p'l<'a> T'a'g<'a> {
```

File: src/tag_search.rs [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                name: "!",
+                file: "change.txt",
+                // _tag: "/*!*"
+            }
+        );
+    }
+
+    #[test]
+    fn test_tag_ord() {
+        let tag = Tag::from_str("!	change.txt	/*!*");
```
Comment (gpanders) [inline src/tag_search.rs:81]:
```suggestion
        let tag = Tag::from_str("!	hope_and_change.txt	/*!*");
```

**Review by gpanders — CHANGES_REQUESTED** [review]

**Comment by Maverun** [comment]
LGTM
also got a sandwich! :sandwich:

**Review by seandewar — CHANGES_REQUESTED** [review]
needs a way to execute arbitrary code :thinking:

**Pull request merged**